atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/ConstraintCheckingTransaction.java

```diff
@@ -27,10 +27,10 @@
     /**
      * Returns the cell values in the key-value service, ignoring the cache of local writes.
      */
-    public Map<Cell, byte[]> getIgnoringLocalWrites(TableReference tableRef, Set<Cell> cells);
+    Map<Cell, byte[]> getIgnoringLocalWrites(TableReference tableRef, Set<Cell> cells);
```

Comment:
These are redundant qualifiers. every method listed in a public interface is public.